packages/client/test/miner/pendingBlock.spec.ts

```diff
@@ -437,10 +440,12 @@ describe('[PendingBlock]', async () => {
 
   it('should exclude missingBlobTx', async () => {
     const gethGenesis = require('../../../block/test/testdata/4844-hardfork.json')
+    let kzg
     try {
-      initKZG(kzg, __dirname + '/../../src/trustedSetups/official.txt')
+      // initKZG(kzg, __dirname + '/../../src/trustedSetups/official.txt')
+      kzg = await createKZG()
     } catch {
-      //no-op
+      // no-op
```

Comment:
This `try catch` seems to be removed at other places? What did it do and should we keep it or remove it altogether?

Reply:
Same as with other tests it seems, `try catch` is still there.